.pre-commit-config.yaml

```diff
@@ -41,7 +41,6 @@ repos:
       entry: python scripts/check_toc_is_complete.py
       language: python
       name: Check all notebooks appear in table of contents
-      pass_filenames: false
```

Comment:
My bad here, I erroneously added this in #4262 :flushed: This script does take file names though (via `argparse`)